packages/discord.js/src/util/Transformers.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const snakeCase = require('lodash.snakecase');
+const transform = require('lodash.transform');
+
+class Transformers extends null {
+  /**
+   * Transforms camel-cased keys into snake cased keys
+   * @param {*} obj The object to transform
+   * @returns {*}
+   */
+  static toSnakeCase(obj = {}) {
+    return transform(obj, (acc, value, key, target) => {
+      const camelKey = Array.isArray(target) ? key : snakeCase(key);
```

Comment:
I don't find this in the docs... what is `target`? And in which cases can it be an array? The [docs](https://lodash.com/docs/4.17.15#transform) are unclear about it.

Reply:
`target` is the original object it can be an array if any props on the object are an array or the object is an array itself.